src/proto/CMakeLists.txt

```
@@ -0,0 +1,54 @@
+find_package(Protobuf REQUIRED)
+
+# Set up the output path.
+set(PROTO_GEN_DIR ${PROJECT_SOURCE_DIR}/build/src)
+set(MESSAGE_DIR ${PROTO_GEN_DIR}/proto)
```

```suggestion
set(PROTO_GEN_DIR ${PROJECT_SOURCE_DIR}/build/src/proto)
```

`MESSAGE_DIR`, `MESSAGE_SRCS`... are misleading, could you update it to `PROTO_DIR`, `PROTO_SRCS`?

---

Done.